luigi_tools/parameter.py

```diff
@@ -180,3 +181,31 @@ class OptionalTupleParameter(OptionalParameter, luigi.TupleParameter):
     """Class to parse optional tuple parameters."""
 
     expected_type = tuple
+
+
+class PathParameter(luigi.Parameter):
+    """Class to parse file path parameters.
+
+    Args:
+        absolute (bool): the given path is converted to an absolute path.
+        create (bool): a folder is automatically created to the given path.
```

Comment:
Does it make sense to rename `create` to `create_parents`? I think that way will be more consistent with the pathlib jargon. https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir

Reply:
Right that's why it's called parents (plural). So it creates all of the parents (complete path).

Reply:
Also this was one of the things I found different in Luigi when I switched from Snakemake. Snakemake was automatically creating the parent folders of the output files so that I wouldn't have to write code to check if they are not created and create them if they are not created.

I am still thinking in the `snakemake` way :))

Reply:
> `/tmp/test/toto/file.test (even file.test is a folder).`

Ah you're right. I was confused, I think for files I was using pathlib like `file.parent.mkdir`. That's why the file was not created.

Reply:
> For me create_parents is more confusing. I would expect it behaves like my_path.parent.mkdir(), not like my_path.mkdir().

Ok I see your point now. Yes `create` makes more sense (since the Path will be a folder)